tests/components/nut/test_config_flow.py

```diff
@@ -113,10 +212,39 @@ def _get_mock_pynutclient(list_vars=None):
                 "username": "test-username",
                 "password": "test-password",
                 "port": 2222,
-                "alias": "ups1",
-                "resources": ["battery.charge"],
             },
         )
 
     assert result2["type"] == "form"
     assert result2["errors"] == {"base": "cannot_connect"}
+
+
+async def test_options_flow(hass):
+    """Test config flow options."""
+
+    config_entry = MockConfigEntry(
+        domain=DOMAIN,
+        unique_id="abcde12345",
+        data=VALID_CONFIG,
+        options={CONF_RESOURCES: ["battery.charge"]},
+    )
+    config_entry.add_to_hass(hass)
+
+    mock_pynut = _get_mock_pynutclient(
+        list_vars={"battery.voltage": "voltage"}, list_ups=["ups1"]
+    )
+
+    with patch(
+        "homeassistant.components.nut.PyNUTClient", return_value=mock_pynut,
+    ), patch("homeassistant.components.nut.async_setup_entry", return_value=True):
+        result = await hass.config_entries.options.async_init(config_entry.entry_id)
+
+        assert result["type"] == data_entry_flow.RESULT_TYPE_FORM
+        assert result["step_id"] == "init"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5329694)